.golangci.yml

```diff
@@ -65,7 +65,7 @@ linters-settings:
   funlen:
     # Checks the number of lines in a function.
     # If lower than 0, disable the check.
-    lines: 200
+    lines: 300
```

Comment:
I think if we have a func this long it usually means we need to break it into smaller pieces for maintenance.

Reply:
Okay, I refactored the code to not be too long. More code for you to review though ;-)